### PR TITLE
Fix member invitation ui spec

### PR DESCRIPTION
**Pull request by josephbaylon**
#### Summary
Fix failing member invitation ui spec. There are behavior changes especially on the **MM-18040 Verify Invite New/Existing Users** test. Not sure if release spreadsheet test cases are up to date but I captured the current behavior from master.

#### Ticket Link
none, failing on daily Cypress test

### CODE REVIEW

File: e2e/cypress/integration/enterprise/guest_accounts/member_invitation_ui_spec.js [hunk 1]
```diff
@@ -203,7 +203,7 @@ describe('Guest Account - Member Invitation Flow', () => {
         invitePeople(user1.email, 1, user1.username);
 
         // * Verify the content and message in next screen
-        verifyInvitationSuccess(user1.username, 'This member has been added to the team.');
+        verifyInvitationSuccess(user1.username, 'An invitation email has been sent.');
```
Comment (prapti) [inline e2e/cypress/integration/enterprise/guest_accounts/member_invitation_ui_spec.js]: The change here is failing for me, whereas the former assertion with "This member has been added to the team" works.

Comment (saturninoabril) [reply e2e/cypress/integration/enterprise/guest_accounts/member_invitation_ui_spec.js]: Seeing that as well
Screen Shot 2020-01-31 at 10 37 00 AM

Comment (josephbaylon) [reply e2e/cypress/integration/enterprise/guest_accounts/member_invitation_ui_spec.js]: Updated

**Pull request merged**